app/clients/salesforce/salesforce_engagement.py

```diff
@@ -37,19 +39,23 @@ def create(
     engagement_id = None
     try:
         if account_id and contact_id:
+            # Default Engagement values, which can be overridden by passing in field_updates
+            field_default_values = {
+                "Name": service.name,
+                "AccountId": account_id,
+                "ContactId": contact_id,
+                "CDS_Opportunity_Number__c": str(service.id),
+                "Notify_Organization_Other__c": get_org_name_from_notes(service.organisation_notes, ORG_NOTES_OTHER_NAME_INDEX),
+                "CloseDate": datetime.today().strftime("%Y-%m-%d"),
+                "RecordTypeId": current_app.config["SALESFORCE_ENGAGEMENT_RECORD_TYPE"],
+                "StageName": ENGAGEMENT_STAGE_TRIAL,
+                "Type": ENGAGEMENT_TYPE,
+                "CDS_Lead_Team__c": ENGAGEMENT_TEAM,
+                "Product_to_Add__c": ENGAGEMENT_PRODUCT,
+            }
+            field_values = field_default_values | field_updates
```

Comment:
I just tried running this code and it works, very cool! I'm more used to `{**field_default_values, **field_updates}`  but this is good too.

Reply:
heh same, I'd always used `{**field_default_values, **field_updates}` this in the past as well